packages/messaging/lib/index.js

```diff
@@ -448,6 +478,11 @@ class FirebaseMessagingModule extends FirebaseModule {
     this._isDeliveryMetricsExportToBigQueryEnabled = enabled;
     return this.native.setDeliveryMetricsExportToBigQuery(enabled);
   }
+
+  async isSupported() {
+    // No-op as web platform isn't implemented
+    return true;
```

Comment:
I think technically we can check on android if play services exists? FCM doesn't work on non-play-services devices as far as I know, and we have that native implementation (with JS wrapper) in app utils

Reply:
I'm not sure on this one. I think I'll see what the consensus is on checking play service availability for android before implementing 😄